.github/workflows/compatibility.yml

```diff
@@ -69,4 +69,4 @@ jobs:
           tools:       composer
           coverage:    none
       # run CI checks
-      - run: bash bin/run-ci-tests.bash
\ No newline at end of file
+      - run: bash bin/run-ci-tests.bash
```

Comment:
Could have sworn I turned this off in Vim. :thinking:

Anyway, none of the other files have newlines at the end, so now it's consistent. :smile:

Reply:
I think this is adding a new line at the end of file? So that's good I think.

Reply:
Oh yeah you're right!